pkg/commands/compute/build_test.go

```diff
@@ -394,16 +289,24 @@ func TestBuildAssemblyScript(t *testing.T) {
 			wantError: "unsupported language foobar",
 		},
 		{
-			name: "AssemblyScript success",
+			name: "successful build",
 			args: args("compute build"),
-			fastlyManifest: `
+			fastlyManifest: fmt.Sprintf(`
 			manifest_version = 2
 			name = "test"
-			language = "assemblyscript"`,
+			language = "assemblyscript"
+
+      [scripts]
+      build = "%s"`, buildScriptAssemblyScript),
 			wantOutputContains: "Built package 'test'",
+			skipWindows:        true,
 		},
 	} {
 		t.Run(testcase.name, func(t *testing.T) {
+			if fstruntime.Windows && testcase.skipWindows {
```

Comment:
There are two tests I've added a "Skip Windows" logic block to, and that's because we use `$()` command substitution to lookup the user's node_modules directory so we can safely execute the npm compiler binaries necessary. The `$()` syntax doesn't exist for `cmd.exe` on Windows. To support Windows would require the tests to be rewritten to use a proprietary `FOR` loop syntax which isn't worth implementing here.

**Side note:** The use of `$()` in the `script.build` of each starter kit is likely going to be an issue for these specific Window users and we'll need to document this somewhere either in DevHub or in each starter kit. That said I believe most Window developers nowadays use Powershell, Cygwin, WSL or some other posix compatible shell and so it shouldn't be a big issue).